Writerside/topics/Quickstart-Up-And-Running.md

```diff
@@ -0,0 +1,463 @@
+    ami = "ami-0f496107db66676ff"
+    instance_type = "t2.small"
+    key_name = "windows-server"
+
+    root_block_device {
+        volume_size = 50
+    }
+
+    vpc_security_group_ids = [var.security_group_id]
+    subnet_id = var.subnets[0].id
+
+    tags = {
+        Name = "${var.workspace_prepend}Windows Server"
+    
+    }
+}
+```
+
+Additionally, we should add additional exposed ports to the security group:
+
+> For the purpose of this tutorial, we can add these to the existing security group, but in production it is a better choice to have 
+> separate security groups for your game servers and to not expose RDP
+
+```
+    ingress {
+        description = "Game Server Traffic"
+        from_port   = 7777
+        to_port     = 7777
+        protocol    = "udp"
+        cidr_blocks = ["0.0.0.0/0"] 
+    }
+
+    ingress {
+        description = "RDP Traffic"
+        from_port   = 3389
+        to_port     = 3389
+        protocol    = "tcp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+    ingress {
+        description = "RDP UDP Traffic"
+        from_port   = 3389
+        to_port     = 3389
+        protocol    = "udp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+```
+
+Next plan and apply your infrastructure
+
+```
+terraform plan -var-file="vars.tfvar" 
+terraform apply -var-file="vars.tfvar"
+```
+
+Once terraform finishes, you can go to the AWS EC2 Console, click your instance, and click "Connect to instance." Under the RDP Tab, download the
+Remote Desktop File. Then click "Get Password" and paste in the private key from the keypair you created previously.
+
+At this point you can connect to the Windows server using RDP.
+
+> Here you can setup your server to run and listen on a certain port on this machine, our example server runs on port 7777.
+
+### Run the Server Sidecar
+
+The server sidecar enables Catena to be aware of your game server without requiring any changes to the code of the game server itself. It is additionally able to make requests on behalf of your game server
+if you so choose. See [Sidecar](Sidecar.md).
+
+First download the Sidecar to the machine, the latest release can be found [here](https://github.com/CatenaTools/sidecar/releases/tag/v0.0.1). Alternatively, clone and build the sidecar.
+
+In the same directory as the sidecar, add a `config.json.` There is an example in the  sidecar repo here: https://github.com/CatenaTools/sidecar/blob/main/config.json
+
+```json
+{
+  "motd": "Hello Catena",
+  "app": {
+    "logging": {
+      "level": "debug",
+      "encoder": "console"
+    },
+    "event_source": {
+      "use": "virtual_server_event_source",
+      "configs": {
+        "catena_api_event_source": {
+          "GrpcListenAddr": "0.0.0.0:8080"
+        },
+        "virtual_server_event_source": {
+          "strategy": "continuous_strategy",
+          "continuous_strategy": {
+            "RequestInterval": "5s"
+          }
+        }
+      }
+    },
+    "resolver": {
```

Comment:
Will the `Sidecar.md` explain some of the concepts, like resolvers?

Reply:
Yes, but it's not written yet, will add some notes